docs/source/en/tasks/object_detection.mdx

```diff
@@ -0,0 +1,585 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Object detection
+
+[[open-in-colab]]
+
+Object detection is a computer vision task of detecting instances of semantic objects that belong to a particular
+class (such as humans, buildings, or cars) in parts of an image. Object detection models receive an image as input and output
+coordinates of the bounding boxes and associated labels of the detected objects. An image can contain multiple objects,
+each with its own bounding box and a label (e.g. it can have a car and a building).
+be present in different parts of an image (e.g. the image can have several cars).
+This task is commonly used in autonomous driving for detecting things like pedestrians, road signs, and traffic lights.
+Other applications include counting objects in images, image search, and more.
+
+ <Tip>
+Check out the <a href="https://huggingface.co/tasks/object-detection">object detection</a> task page to learn about use cases,
+models, metrics, and datasets associated with this task.
+ </Tip>
+
+In this guide, you will learn how to:
+
+ 1. Finetune [DETR](https://huggingface.co/docs/transformers/model_doc/detr), a model that combines a convolutional
```

Comment:
```suggestion
 1. Fine-tune [DETR](https://huggingface.co/docs/transformers/model_doc/detr), a model that combines a convolutional
```

Reply:
In an opensource repo like Transformers, it's going to be very tedious to maintain a consistent spelling since we have so many contributors (for this or pre-trained/pretrained). The positive impact of forcing one spelling is exactly zero since both are perfectly understandable, while taking time from reviewers/maintainers, so overall a negative impact.